src/app/(app)/specialist/[id]/page.js

```diff
@@ -22,9 +35,12 @@ export async function generateMetadata({ params }) {
 
 export default async function Page({ params }) {
   const { id } = params;
-  const specialist = await getSpecialistById({ id });
+  const cardStyle = 'mx-auto my-6 max-w-[900px] px-4 md:my-10 lg:px-0';
 
-  return (
-    <CardSpecialist specialist={specialist} extended className="mx-auto my-6 max-w-[900px] px-4 md:my-10 lg:px-0" />
-  );
+  const specialist = await getSpecialistById({ id });
```

Comment:
🟠 maybe on click beside id we could put provider type too, to avoid blindly pulling two entities ? then we could take type ( org or spec ) and id, and then pull exactly what we need from first time